tripleblindmarket/covid-safe-paths
# Shifted About device from Legal page to About page

## Description

Shifted About device from Legal page to About page
Fixes #471 

## How to test

Visit Legal page and verify whether the About device info is empty 
Visit About page and verify whether the About device info is present

![Screenshot_2020-04-14-08-45-04-524_org pathcheck covidsafepaths](https://user-images.githubusercontent.com/25605193/79182547-b0514b80-7e2c-11ea-86a1-c15d1063127f.jpg)
